js/views/playingScreen.js

```diff
@@ -19,9 +19,15 @@ var PlayingScreen = React.createClass({
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
     this.browserSupportsTouch = this.props.controller.state.browserSupportsTouch;
+    this.isVideo360 = this.props.controller.state.isVideo360;
```

Comment:
I know that the older code is already doing this, but storing props in local state is an anti-pattern. Could we use the value from props directly?